### PR TITLE
Disable enter key connection if invalid and changed invalid/valid icons to SVG

### PR DESCRIPTION
**Pull request by martinfrancois**
Fixes issue #120 and we now have prettier looking icons!

### CODE REVIEW

**Review by rmosimann — APPROVED** [review]
look is ok.

File: resources/css/styles.css [hunk 1]
```diff
@@ -202,6 +202,10 @@
  *                                                                         *
  **************************************************************************/
 
+#validationImgView {
+
+}
```
Comment (l-marchesi) [inline resources/css/styles.css]: You did not specify anything in here... so if there is no styling planned (as I assume so), you can delete this here.

File: src/ch/imedias/rsccfx/view/RsccSupportView.java [hunk 2]
```diff
@@ -128,6 +124,9 @@ private void layoutForm() {
     VBox.setVgrow(startServiceInnerPane, Priority.ALWAYS);
     startServiceInnerPane.getStyleClass().add("contentSupport");
 
+    validationImgView.setId("validationImgView");
```
Comment (l-marchesi) [inline src/ch/imedias/rsccfx/view/RsccSupportView.java]: Same as my comment in CSS. If you don't plan to use the CSS, you can delete this file. If you think that consistency needs to be applied, leave it in but write a comment in the css.

**Review by l-marchesi — APPROVED** [review]
Decide about the CSS styling, otherwise: Go ahead and merge it :+1:

File: resources/css/styles.css (hunk 1 above)
Comment (martinfrancois) [reply resources/css/styles.css]: well seen, thanks!

File: src/ch/imedias/rsccfx/view/RsccSupportView.java (hunk 2 above)
Comment (martinfrancois) [reply src/ch/imedias/rsccfx/view/RsccSupportView.java]: same as above, thanks!

**Pull request merged**